### PR TITLE
All player cards from TSH pack

### PR DESCRIPTION
**Pull request by zzorba**
Spoilers so don't merge yet, but the street date looks like Dec 13.

### CODE REVIEW

File: pack/tde/tsh.json [hunk 1]
```diff
@@ -1,4 +1,55 @@
...
+		"is_unique": true,
+		"name": "Tetsuo Mori",
+		"pack_code": "tsh",
+		"position": 155,
+		"quantity": 2,
+		"skill_intellect": 1,
+		"sanity": 2,
+		"slot": "Ally",
+		"subname": "Too Noble for His Own Good",
+		"text": "Tetsuo Mori may be assigned damage and/or horror dealt to other investigators at your location.\n[reaction] When Tetsuo Mori is defeated: Choose an investigator at your location. That investigator searches either the their discard pile or the top 9 cards of their deck for an [[Item]] asset and adds it to their hand. Shuffle their deck if it is searched.",
```
Comment (drawntotheflame) [inline pack/tde/tsh.json]: 'searches either the their' <- delete 'the'

File: pack/tde/tsh.json [hunk 2]
```diff
@@ -29,10 +97,66 @@
 		"quantity": 2,
 		"skill_combat": 1,
 		"text": "Play only during one of your turns in which you defeated enemies with a total of 6 or more health.\nAdd \"Let God sort them out...\" to the victory display and immediately end your turn. When earning experience during resolution of this scenario, you earn 1 additional experience.",
```
Comment (drawntotheflame) [inline pack/tde/tsh.json]: 'during resolution' <- change to 'during the resolution'

File: pack/tde/tsh.json [hunk 3]
```diff
@@ -29,10 +97,66 @@
...
+		"cost": 3,
+		"deck_limit": 2,
+		"faction_code": "rogue",
+		"illustrator": "Stephen Somers",
+		"name": "Swift Reload",
+		"pack_code": "tsh",
+		"position": 161,
+		"quantity": 2,
+		"skill_agility": 1,
+		"text": "Fast. Play only during your turn.\nChoose a [[Firearm]] asset you control with fewer ammo tokens than its uses (X) value. Place ammon on that asset until it has ammo equal to its uses (X) value.",
```
Comment (drawntotheflame) [inline pack/tde/tsh.json]: 'Place ammon' <- change to 'Place ammo'.

File: pack/tde/tsh.json [hunk 4]
```diff
@@ -50,6 +174,38 @@
...
+		"quantity": 2,
+		"skill_combat": 1,
+		"text": "Max 1 committed per skill test.\nWhile Brute Force is committed during a basic fight action, it gains [combat][combat] and the text: \"If this test is successful by 2 or more, this attack deals +2 damage.\"",
+		"traits": "Innate. Developed.",
+		"type_code": "skill",
+		"xp": 1
+	},
 	{
 		"code": "06167",
 		"cost": null,
```
Comment (drawntotheflame) [inline pack/tde/tsh.json]: add  after 'faction code': 

"flavor": "\"Unassuming\" doesn't mean you're unprepared.",

**Comment by zzorba** [comment]
All fixed.  @Kamalisk this can be merged now.

**Pull request merged**